### ISSUE TITLE
Table scrolling is lagging when there are 20 columns

### ISSUE THREAD

**Issue by FrominXu** [rsuite/rsuite#2617]
### What version of rsuite are you using?

5.16.1

### What version of React are you using?

17

### Describe the Bug

try this 
https://codesandbox.io/s/rsuite-table-test-c1ygm5?file=/src/Table/index.js

I think the reason is on calculating.  the horizontal scrolling is not virtualized, but it is lagging too, so strange. 

ps: I tried the same case in Ag-Grid, it is smooth.

**Comment by simonguo** [comment]
As you can see, rsuite table does not add virtualized to horizontal scrolling. If you need to display more columns, it may not be suitable to use rsuite table temporarily.

**Comment by FrominXu** [comment]
> As you can see, rsuite table does not add virtualized to horizontal scrolling. If you need to display more columns, it may not be suitable to use rsuite table temporarily.

yes, I know rsuite table does not add virtualized to horizontal scrolling. What I mean is no virtualized means all columns cell DOM elements are mounted, why it is still slagging when not need to re-render?
